crates/bevy_ui/src/flex/convert.rs

```diff
@@ -1,99 +1,77 @@
-use taffy::style::LengthPercentageAuto;
+use bevy_math::Vec2;
 
 use crate::{
     AlignContent, AlignItems, AlignSelf, Display, FlexDirection, FlexWrap, JustifyContent,
     PositionType, Size, Style, UiRect, Val,
 };
 
 impl Val {
-    fn scaled(self, scale_factor: f64) -> Self {
+    fn scaled(self, scale_factor: f64, physical_size: Vec2) -> Self {
         match self {
             Val::Auto => Val::Auto,
             Val::Percent(value) => Val::Percent(value),
             Val::Px(value) => Val::Px((scale_factor * value as f64) as f32),
+            Val::VMin(value) => Val::Px(physical_size.x.min(physical_size.y) * value / 100.),
+            Val::VMax(value) => Val::Px(physical_size.x.max(physical_size.y) * value / 100.),
```

Comment:
It would probably make sense to do the `max` and `min` computation at a higher level and pass them in. Otherwise you're doing them for every style that uses them whereas you only need to do them once.